exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StorageResources.java

```diff
@@ -204,6 +204,10 @@ public JsonResult createOrUpdatePluginJSON(PluginConfigWrapper plugin) {
   @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
   @Produces(MediaType.APPLICATION_JSON)
   public JsonResult createOrUpdatePlugin(@FormParam("name") String name, @FormParam("config") String storagePluginConfig) {
+    name = name.trim();
```

Comment:
Can name be null? Just checking to sure we won't get NPE here.

Reply:
Under normal circumstances - no. We use a form with two fields to send a POST request, you can leave some of them empty, but they will be initialized anyway. 
You can brake it intentionally of course, for example by deletion of a form filed element via Developer tools or using Postman to send a malformed request.